src/app-root.ts

```diff
@@ -377,6 +387,12 @@ export class AppRoot extends LitElement {
           @searchTypeChanged=${this.searchTypeChanged}
           @manageModeChanged=${this.manageModeChanged}
         >
+          ${this.toggleSlots
+            ? html`<div slot="sort-slot-left">Sort Slot</div>`
+            : nothing}
+          ${this.toggleSlots
+            ? html`<div slot="facet-top-slot">Facet Slot</div>`
+            : nothing}
```

Comment:
Thanks for the demo additions to show these clearly!

Reply:
There is one more slots for list header `cb-top-slot`. lets add this one as well here.